tests/build_/test_build_request.py

```diff
@@ -1044,8 +1044,17 @@ def test_render_optional_plugins(self, tmpdir):
         # Only one version defined so far
         DEFAULT_ARRANGEMENT_VERSION,
     ])
+    @pytest.mark.parametrize('build_image', [
```

Comment:
I think this should be more explicit on what the expected behavior is. 
This unit test is re-implementing logic that is being tested.
I think a stronger test would explicitly specify what the worker_build_image value should be depending on the values of `build_image` and `build_imagestream`.
Something along the lines of:

```python
@pytest.mark.parametrize(['build_image', 'build_imagestream', 'worker_build_image'], [
    ('fedora:latest', None, 'fedora:latest'),
    ...
])
```